CHANGELOG.md

```diff
@@ -42,6 +42,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - `GET /resources` request with non-numeric delimiter (limit or offset) now
   returns `Error 422 Unprocessable Entity` instead of `Error 500`.
   [cyberark/conjur#1997](https://github.com/cyberark/conjur/issues/1997)
+- `POST /host_factory_tokens` request with invalid ip address or CIDR range of `cidr` parameter
```

Comment:
@shulifink

Reply:
How about
`POST /host_factory_tokens` request which has invalid ip address or CIDR range in `cidr` parameter

Reply:
@liavyona not sure the context of the text - confused by the orignal and @InbalZilberman's comment

In the meantime, IP is always caps